recipe/meta.yaml

```diff
@@ -32,29 +32,31 @@ requirements:
     - cachey >=0.2.1
     - certifi >=2020.6.20
     - dask >=2.15.0
-    - imageio >=2.5.0
+    - imageio >=2.5.0,!=2.11.0
     - importlib-metadata >=1.5.0  # not needed for py>37 but keeping for noarch
     - jsonschema >=3.2.0
-    - magicgui >=0.3.0
+    - magicgui >=0.3.3
     - napari-console >=0.0.4
     - napari-plugin-engine >=0.1.9
     - napari-svg >=0.1.4
+    - npe2 >=0.1.1
     - numpy >=1.18.5
     - numpydoc >=0.9.2
+    - pandas >=1.1.0
     - pillow
     - pint >=0.17
     - psutil >=5.0
     - pyopengl >=3.1.0
     - pyyaml >=5.1
     - pydantic >=1.8.1
     - qtpy >=1.7.0
-    - scipy >=1.2.0
-    - superqt >=0.2.2
+    - scipy >=1.4.0
+    - superqt >=0.2.5
     - tifffile >=2020.2.16
     - typing_extensions
     - toolz >=0.10.0
     - tqdm >=4.56.0
-    - vispy >=0.6.4
+    - vispy >=0.9.4
     - wrapt >=1.11.1
 
     # additional dependencies for convenience in conda-forge
```

Comment:
At least that's what `testing` [suggests](https://github.com/napari/napari/blob/main/setup.cfg#L108) in our setup.cfg

Reply:
0.19.0 gave us a bug in testing because of a change in the way they fetched data from pooch.  it isn't used in the napari API (just for our tests).  I'm not sure if it warrants complete incompatibility here,  @jni?

Reply:
I don't think it matters, let's ignore it! Thanks for flagging @jaimergp!